style.css

```diff
@@ -0,0 +1,901 @@
+    background-image: url(img/bg-headear.png);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  .main-section-article {
+    margin-left: 3rem;
+    height: 100vh;
+    margin-top: 1rem;
+  }
+
+  .main-section-article h1 {
+    margin-top: 4rem;
+    font-size: 25px;
+    font-weight: 600;
+    text-transform: uppercase;
+    margin-left: 1rem;
+    margin-right: 1rem;
+  }
+
+  .main-section-article h2 {
+    font-size: 43px;
+    font-weight: 900;
+    width: 60%;
+    color: #ec5342;
+    margin-top: -1rem;
+    margin-left: 1rem;
+    margin-right: 1rem;
+    margin-bottom: 4rem;
+  }
+
+  .article-1 {
+    background-color: whitesmoke;
+    width: 54%;
+    height: 25vh;
+    margin-top: 1rem;
+    margin-left: 1rem;
+    margin-right: 1rem;
+    margin-bottom: 1rem;
+    text-align: left;
+  }
+
+  .nav-list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    text-align: center;
+    width: 60%;
+    margin: 0 2rem;
+  }
+
+  .date {
+    margin-left: 1rem;
+  }
+
+  .place {
+    margin-left: 1rem;
+    font-size: x-large;
+    font-weight: 400;
+  }
+
+  .portfolio-link-a-nav {
+    color: #262930;
+  }
+
+  .portfolio-link-a-nav:hover {
+    color: #ec5342;
+  }
+
+  .portfolio-link-nav .active {
+    color: #ec5342;
+  }
+
+  .portfolio-link-nav .active:hover {
+    color: #262930;
+  }
+
+  .summit {
+    text-align: center;
+    color: #ec5342;
+    border: 3px solid #ec5342;
+    display: block;
+    width: 5rem;
+    padding: 0.3rem;
+  }
+
+  /* /======Second Section============== */
+
+  .second-section {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 70vh;
+    flex-wrap: wrap;
+  }
+
+  .second-section .container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 90%;
+  }
+
+  .second-section .card {
+    flex-direction: column;
+    height: 26vh;
+    justify-content: space-evenly;
+    align-items: center;
+    padding: 1rem;
+  }
+
+  .third-section .card {
+    width: 40%;
+    height: 20vw;
+    font-size: small;
+    padding: 1rem;
+  }
+
+  .second-section .card:hover {
+    border: 1px solid white;
+    opacity: 1;
+  }
+
+  .second-section h2 {
+    width: 100%;
+    margin: 1rem;
+  }
+
+  .hr-about-1 {
+    width: 9%;
+    margin-top: -2rem;
+  }
+
+  .card .content {
+    width: 100%;
+    font-size: small;
+    font-weight: 400;
+    text-align: center;
+    padding: 0.3rem;
+  }
+
+  .card img {
+    width: 50%;
+  }
+
+  .card .title {
+    font-size: medium;
+    font-weight: 400;
+    width: 100%;
+  }
+
+  a,
+  a:hover,
+  a:focus,
+  a:active,
+  a:visited {
+    text-decoration: none;
+  }
+
+  .second-section a {
+    display: block;
+    color: white;
+    text-decoration: underline;
+    text-transform: uppercase;
+    margin: 2rem;
+    transition: all 0.7s 0s ease;
+  }
```

Comment:
- [ ] Please fix the top navbar by adding some padding and changing the color.
## Your design:
![image](https://user-images.githubusercontent.com/92215878/192030351-77af01af-892c-49cf-a84e-c2983e79ba16.png)

## Given design:
![image](https://user-images.githubusercontent.com/92215878/192030422-a4cf6c00-2d2d-49aa-b315-696c38816b52.png)

Reply:
@jasonfelice  may you help me with that, I try a lot of images, png, SVG, and material icons, and I can't fix it

Reply:
@jasonfelice Thank you, I fixed it💪